cli/public_key/cli.py

```diff
@@ -28,7 +34,39 @@ def public_key_commands():
     pass
 
 
-@click.option('--address', type=str, required=True, help=ADDRESS_ARGUMENT_HELP_MESSAGE)
+@click.option('--address', type=str, required=True, help=PUBLIC_KEY_ADDRESS_ARGUMENT_HELP_MESSAGE)
+@click.option('--node-url', type=str, required=False, help=NODE_URL_ARGUMENT_HELP_MESSAGE, default=default_node_url())
+@public_key_commands.command('get-info')
+def get_public_key_info(address, node_url):
+    """
+    Get information about public key address by public key address.
```

Comment:
The same about you get information about the public key, not its address.

Reply:
@dmytrostriletskyi, done.